circle.yml

```diff
@@ -6,8 +10,31 @@ dependencies:
     - echo y | android update sdk --no-ui -a --filter extra-google-google_play_services
     - echo y | android update sdk --no-ui -a --filter extra-google-m2repository
     - echo y | android update sdk --no-ui -a --filter extra-android-m2repository
+    - echo y | android update sdk --no-ui --all --filter sys-img-armeabi-v7a-google_apis-24
+    - echo no | android create avd -n circleci-android24-googleapis -t 'android-24' --abi google_apis/armeabi-v7a
+    - |
+      wget "https://dl-ssl.google.com/android/repository/sys-img/google_apis/armeabi-v7a-24_r08.zip"
+      unzip armeabi-v7a-24_r08.zip
+      mv armeabi-v7a/* /usr/local/android-sdk-linux/system-images/android-24/google_apis/armeabi-v7a
+      rm armeabi-v7a-24_r08.zip
+
+machine:
+  environment:
+    QEMU_AUDIO_DRV: none
 
 test:
+  override:
+    - emulator -avd circleci-android24-googleapis -no-window -gpu off:
+        background: true
+        parallel: true
+    - circle-android wait-for-boot
+    - adb devices
+    - adb shell input keyevent 82
+    - ./gradlew core:connectedAndroidTest -Dorg.gradle.parallel=false -PdisablePreDex:
+        timeout: 720
+    - cp -r app/build/outputs $CIRCLE_ARTIFACTS
+    - cp -r app/build/outputs/androidTest-results/* $CIRCLE_TEST_REPORTS
+    - ./gradlew test
```

Comment:
HEY, this is mine 😢 
So not only do you steal my Javadocs, but now you're taking credit of my code to?
How low the Android Team has sunk since @bryant1410 left